### PR TITLE
ir.lv2: 1.2.2 -> 1.2.3

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/ir.lv2/default.nix [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
   src = fetchurl {
-    url = "http://factorial.hu/system/files/${name}.tar.gz";
-    sha256 = "17a6h2mv9xv41jpbx6bdakkngin4kqzh2v67l4076ddq609k5a7v";
+    url = "https://github.com/tomszilagyi/ir.lv2/archive/${version}.tar.gz";
+    sha256 = "133pfzs82dd60pnxk99ig3dwwkczir2w55zcflkhs8kw10p34m99";
   };
 
-  buildInputs = [ fftw gtk lv2 libsamplerate libsndfile pkgconfig zita-convolver ];
+  buildInputs = [ fftw gtk2 lv2 libsamplerate libsndfile pkgconfig zita-convolver ];
```
Comment (Mic92) [inline pkgs/applications/audio/ir.lv2/default.nix]: Can you also move `pkgconfig` to `nativeBuildInputs`?

File: pkgs/applications/audio/ir.lv2/default.nix [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 stdenv.mkDerivation rec {
   name = "ir.lv2-${version}";
-  version = "1.2.2";
+  version = "1.2.3";
 
   src = fetchurl {
-    url = "http://factorial.hu/system/files/${name}.tar.gz";
-    sha256 = "17a6h2mv9xv41jpbx6bdakkngin4kqzh2v67l4076ddq609k5a7v";
+    url = "https://github.com/tomszilagyi/ir.lv2/archive/${version}.tar.gz";
```
Comment (Mic92) [inline pkgs/applications/audio/ir.lv2/default.nix]: `fetchFromGitHub` is the preferred option here.

File: pkgs/applications/audio/ir.lv2/default.nix [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+    sha256 = "133pfzs82dd60pnxk99ig3dwwkczir2w55zcflkhs8kw10p34m99";
   };
 
-  buildInputs = [ fftw gtk lv2 libsamplerate libsndfile pkgconfig zita-convolver ];
+  buildInputs = [ fftw gtk2 lv2 libsamplerate libsndfile pkgconfig zita-convolver ];
 
   buildPhase = ''
```
Comment (Mic92) [inline pkgs/applications/audio/ir.lv2/default.nix]: I would use postBuild instead:

```
postBuild = "make convert4chan"
```

**Comment by magnetophon** [comment]
@Mic92 Thanks for the feedback.

**Comment by magnetophon** [comment]
@fpletz Thanks for all those quick merges!

**Comment by fpletz** [comment]
@magnetophon Sure, tomorrow is the 17.03 branch-off, so we have to get in all the uncontroversial stuff. Also, you are the maintainer of these packages and travis was green… so how can I resist? :smile:

Thanks for your contributions! :beers:

**Comment by magnetophon** [comment]
@fpletz aha, good to know.

Would [FFTW 3.3.5 -> 3.3.6](http://fftw.org/release-notes.html0) fall under "uncontroversial stuff"?
It's just bugfixes, but it will cause a mass-rebuild.

**Comment by fpletz** [comment]
The changelog looks innocent enough and just fixes stuff, let's do it!

**Pull request merged**